packages/next/src/build/index.ts

```diff
@@ -2989,8 +2989,7 @@ export default async function build(
           }
 
           // remove temporary export folder
-          await recursiveDelete(exportOptions.outdir)
-          await fs.rmdir(exportOptions.outdir)
+          await fs.rm(exportOptions.outdir, { recursive: true, force: true })
```

Comment:
@ijjk I was trying to figure out why this was using two deletes and it looks like this goes back to 2019:

- https://github.com/vercel/next.js/pull/7293

Does the `fs.rm()` solution solve this use case now?